### DISCUSSION TITLE
How to overlay modal dismiss button on top of image instead of using modal title?

### DISCUSSION THREAD

**Discussion by dosstx** [twbs/bootstrap#38126] [Q&A] [answered]
I am using Bootstrap5 Modal as an image "lightbox" for showing full screen images. Is there a way to overlay the dismiss/close button on the image instead of using the default modal title component? Otherwise I have empty spacing for the title that is not needed on top of the image. Hope this makes sense. Thanks!

**Comment by ghost** [comment, marked as answer]
Yes, you can overlay the dismiss/close button on the image in Bootstrap 5 Modal by customizing the modal header. Instead of using the default modal title component, you can create a custom header that includes the close button and positions it over the image.

Here's an example of how you can achieve this:
```html
<div class="modal fade" id="imageModal" tabindex="-1" aria-labelledby="imageModalLabel" aria-hidden="true">
  <div class="modal-dialog modal-fullscreen">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="btn-close position-absolute top-0 end-0 m-3" data-bs-dismiss="modal" aria-label="Close"></button>
      </div>
      <div class="modal-body p-0">
        <img src="your-image-url" class="img-fluid" alt="Your image">
      </div>
    </div>
  </div>
</div>
```

In this example, we've created a custom modal header by using a close button with the `btn-close` class and positioning it absolutely with the `position-absolute` class and the utility classes `top-0` and `end-0` to place it at the top right corner of the modal. We've also added some margin with the `m-3` class to give some space around the button.

In the modal body, we've added the image with the `img-fluid` class to make it responsive and take up the full width of the modal.

By removing the default modal title component, you can avoid the empty spacing on top of the image.

I hope this helps!
